src/TableFooter.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import classSet from 'classnames';
+
+class TableHeader extends Component {
```

Comment:
If its a tablefooter than class name should be `TableFooter` rather than `TableHeader`